### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values-it/conversationlist.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#186]
```
update translation

added some missing part, to be finished, missing more or less 300 rows yet
```

Original issue reported on code.google.com by `liogiu@gmail.com` on 17 Apr 2011 at 5:08

Attachments:
- [conversationlist.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-186/comment-0/conversationlist.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Great. Thanks again liogiu. Committed as part of r74.
```

Original comment by `oskar.wi...@gmail.com` on 4 May 2011 at 5:38
- Changed state: **Ready**
- Added labels: **Type-Other**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 4 May 2011 at 5:41
- Added labels: **Type-Translation**
- Removed labels: **Type-Other**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:52
- Changed state: **Done**
